cmd/generate/config/rules/artifactory.go

```diff
@@ -0,0 +1,42 @@
+package rules
+
+import (
+	"github.com/zricethezav/gitleaks/v8/cmd/generate/config/utils"
+	"github.com/zricethezav/gitleaks/v8/cmd/generate/secrets"
+	"github.com/zricethezav/gitleaks/v8/config"
+	"github.com/zricethezav/gitleaks/v8/regexp"
+)
+
+func ArtifactoryApiKey() *config.Rule {
+	// define rule
+	r := config.Rule{
+		RuleID:      "artifactory-api-key",
+		Description: "Detected an Artifactory api key, posing a risk unauthorized access to the central repository.",
+		Regex:       regexp.MustCompile(`AKCp[A-Za-z0-9]{69}`),
+		Entropy:     3,
+		Keywords:    []string{"AKCp"},
+	}
+
+	// validate
+	tps := []string{
+		"artifactoryApiKey := \"AKCp" + secrets.NewSecret(utils.Hex("69")) + "\"",
+	}
+	return utils.Validate(r, tps, nil)
```

Comment:
Are you able to add a couple false-positive examples? 

Examples from my testing include:
- https://github.com/samsmithnz/Big10GBRepo/blob/f0d605dc2aba908e7b3510ed2506ad2dc39ea4bb/21/mKALw.txt
- https://github.com/jncdke/wzry-crack-2025-05-27/blob/edc6bb1107da15dc9e2f54eb9967345586211761/RT/4.14.186.sh#L1386
- https://github.com/OdysMour/ds-frontend/blob/decacb8606e43ba30943f7541a0b9194ecfe39a9/bun.lock#L696